packages/ui/src/components/va-modal/hooks/useModal.ts

```diff
@@ -13,8 +13,9 @@ export const useModal = () => {
   /**
    * @param options can be message string or options object
    */
+  let modalInstance: any
```

Comment:
There are cases when two modals can be opened at the same time, then close method will close only last modal. Take a look on `useToast` implementation.

Reply:
i'm learning "toast.ts". It's difficult but i undesrstand what were you talking about

Сan i use getGlobal().vaModalInstances.push(vNode) in method 'createModalInstance' ?
But without getGlobal(), because:

> @deprecated — why can't we just use some local object instead?
> 'getGlobal' is deprecated.ts(6385)